docs/5-operator-manual/0-set-up-and-install.md

```diff
@@ -7,50 +7,54 @@
 
 <!-- TODO: @JPERRY Look at how other tools/apps do their instillation instructions -->
 
-Until we get Zarf into common package managers, you can install the Zarf CLI by:
+In order to install the Zarf CLI, which is not yet available in common Linux package managers, you may follow the steps below:
 
-1. Downloading the latest release version for your machine from our [GitHub release page](https://github.com/defenseunicorns/zarf/releases).
-2. Move the downloaded file onto your path. This can be done in your terminal with the command `mv ~/Downloads/{DOWNLOADED_RELEASE_FILE} /usr/local/bin/zarf`
-3. Test out the CLI within your terminal with the command `zarf -version`. The version you downloaded from GitHub should print to your terminal.
+1. Download the latest release version for your machine from our [GitHub release page](https://github.com/defenseunicorns/zarf/releases).
+2. Transfer the downloaded file onto your path. This can be executed in your terminal with the command `mv ~/Downloads/{DOWNLOADED_RELEASE_FILE} /usr/local/bin/zarf`.
+3. Verify the installation by testing the CLI within your terminal with the command `zarf -version`. If the installation is successful, the version of Zarf CLI that you have downloaded from GitHub should be displayed in your terminal.
+:::note
```

Comment:
```suggestion

:::note
```

This :::note actually needs an extra space
![image](https://user-images.githubusercontent.com/3977569/232901555-d08a937f-6d16-422a-91cb-956451a8b230.png)